schema/schemaspec/spec.go

```diff
@@ -43,6 +46,28 @@ type (
 	Ref struct {
 		V string
 	}
+
+	// File reperesents an Atlas DDL configuration file.
```

Comment:
Can you add more detail here for the reader - what is attribute what is resource?